docs/source/knowledge_base.py

```diff
@@ -0,0 +1,50 @@
+import os
+from pathlib import Path
+from typing import List
+
+FILE_DIR = os.path.dirname(__file__)
+GEN_DIR = "reference/generated/knowledge_base"
+
+
+def _get_file_content(name: str) -> List[str]:
+    """
+    To include markdown files from the knowledge graph in Sphinx, these are linked using the rst include command. This
```

Comment:
The proper way of writing docstrings should be without a new line after the triple quote, and a short description first
```python
def fun():
    """This function does X.

    A way more elaborate explanation.
    """
    return foo
```

Reply:
You are absolutely right, thanks!